src/Nancy.Tests/Unit/ResponseExtensionsFixture.cs

```diff
@@ -48,10 +47,11 @@ public void Should_use_filename_and_content_type_for_attachments_from_file_respo
             var assemblyPath =
                 Path.GetDirectoryName(this.GetType().Assembly.Location);
 
-            GenericFileResponse.SafePaths = new List<string> {assemblyPath};
+            var environment = new DefaultNancyEnvironment();
+            environment.AddValue(new SafePathConfiguration(new[]{ assemblyPath }));

```

Comment:
You can do `environment.SafePaths(assemblyPath)` instead of `environment.SafePaths(new[] { assemblyPath })` :smile: